modules/claims_api/spec/fixtures/form_526_no_flashes_json_api.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
This seems odd.  🤔 
Wasn't this file from another recent PR and we named it `form_526_no_flashes_no_special_issues.json`?